Package.swift

```diff
@@ -53,7 +53,7 @@ let package = Package(
         Target(
             /** Generates Xcode projects */
             name: "Xcodeproj",
-            dependencies: ["PackageType"]),
+            dependencies: ["PackageType", "Build"]),
```

Comment:
This doesn't seem like a good layering. I wouldn't expect the Xcode projection generation to depend on the `Build` module, and would refactor things to avoid that. Conceptually, the Xcode project generator and Build system should be peers which depend on a common model.

If we need a new module to hold "build-related" things which are not directly `PackageType` model objects we should introduce it.